google-cloud-tools-plugin/src/com/google/cloud/tools/intellij/apis/GoogleCloudApiManagementPanel.java

```diff
@@ -0,0 +1,286 @@
+      // TODO(eshaul) query the IAM API to get the actual status of the API
+      return false;
+    }
+
+    @Override
+    public int getRowCount() {
+      return librariesToEnabledStatus.size();
+    }
+
+    @Override
+    public int getColumnCount() {
+      return 2;
+    }
+
+    @Override
+    public String getColumnName(int columnIndex) {
+      return null;
+    }
+
+    @Override
+    public Class<?> getColumnClass(int columnIndex) {
+      switch (columnIndex) {
+        case CLOUD_API_NAME_COL:
+          return CloudLibrary.class;
+        case CLOUD_API_ENABLEMENT_COL:
+          return Boolean.class;
+        default:
+          throw new IndexOutOfBoundsException();
+      }
+    }
+
+    @Override
+    public boolean isCellEditable(int rowIndex, int columnIndex) {
+      return false;
+    }
+
+    @Override
+    public Object getValueAt(int rowIndex, int columnIndex) {
+      switch (columnIndex) {
+        case CLOUD_API_NAME_COL:
+          return librariesToEnabledStatus.keySet().toArray()[rowIndex];
+        case CLOUD_API_ENABLEMENT_COL:
+          return librariesToEnabledStatus.values().toArray()[rowIndex];
+        default:
+          throw new IndexOutOfBoundsException();
+      }
+    }
+
+    @Override
+    public void setValueAt(Object aValue, int rowIndex, int columnIndex) {}
+
+    @Override
+    public void addTableModelListener(TableModelListener l) {}
+
+    @Override
+    public void removeTableModelListener(TableModelListener l) {}
+  }
+
+  /**
+   * Custom cell renderer to handle display of the enablement column of the API enablement table. If
+   * the API is not enabled on GCP, then a button is rendered allowing the user to enable the API
+   * from the IDE.
+   */
+  private static final class CloudApiEnablementRenderer extends DefaultTableCellRenderer {
+
+    private final JButton enableButton =
+        new JButton(GctBundle.message("cloud.apis.management.enable.button"));
+
+    CloudApiEnablementRenderer() {
+      super();
+
+      setOpaque(false);
+      enableButton.setOpaque(false);
+    }
+
+    @Override
+    public Component getTableCellRendererComponent(
+        JTable table, Object value, boolean isSelected, boolean hasFocus, int row, int column) {
+      Couple<Color> colors = UIUtil.getCellColors(table, isSelected, row, column);
+      setForeground(colors.getFirst());
+      setBackground(table.getBackground());
+
+      JComponent component =
+          (JComponent)
+              super.getTableCellRendererComponent(table, value, isSelected, hasFocus, row, column);
+
+      if (column != CLOUD_API_ENABLEMENT_COL) {
+        return component;
+      }
+
+      setHorizontalAlignment(CENTER);
+      boolean isEnabled;
+      if (value instanceof String) {
+        isEnabled = Boolean.parseBoolean((String) value);
+      } else {
+        isEnabled = (Boolean) value;
+      }
+
+      if (isEnabled) {
+
+        component.setBorder(IdeBorderFactory.createEmptyBorder());
```

Comment:
Probably we can try the same idea as for another renderer - pass `isSelected` and `hasFocus` as always false and remove the explicit border call.